eng/pipelines/eng-common-sync.yml

```diff
@@ -15,15 +15,22 @@ parameters:
 - name: Repos
   type: object
   default:
-    - azure-sdk-for-android
-    - azure-sdk-for-c
-    - azure-sdk-for-cpp
-    - azure-sdk-for-go
-    - azure-sdk-for-ios
-    - azure-sdk-for-java
-    - azure-sdk-for-js
-    - azure-sdk-for-net
-    - azure-sdk-for-python
+    azure-sdk-for-android:
+    azure-sdk-for-c:
+    azure-sdk-for-cpp:
+    azure-sdk-for-go:
+    azure-sdk-for-ios:
+    azure-sdk-for-java:
+      definition-id: 731
```

Comment:
Instead of hard-coding these 2 should we just consider a list of id's and we just trigger them all?

Reply:
I don't understand what you mean here.

Reply:
```
azure-sdk-for-java:
   - 1736
   - 2106
```
I was think we can just provide a list something like that so instead of only supporting one build and test id we could just take a list of build ids and loop over them and queue all of them.

Reply:
@chidozieononiwu did you see this feedback?